oshmem/shmem/c/shmem_lock.c

```diff
@@ -269,20 +269,24 @@ static uint64_t shmem_lock_cswap(void *target,
     uint64_t prev_value = 0;
 
     if (target_size == 8) {
-        MCA_ATOMIC_CALL(cswap(oshmem_ctx_default, target, (void*)&prev_value, cond, value, target_size, pe));
+        MCA_ATOMIC_CALL(cswap(oshmem_ctx_default, target, (void*)&prev_value,
+			      cond, value, target_size, pe));
```

Comment:
pls consider pushing changes in this file in a separate PR, as they are not relavant for the new locking algo

Reply:
Removed unwanted changes from this PR.